data/json/professions.json

```diff
@@ -2327,6 +2343,40 @@
       "female": [ "bra", "panties" ]
     }
   },
+  {
+    "type": "profession",
+    "ident": "millionaire",
+    "name": "Millionaire",
+    "description": "You had a fortune, whether inherited, built by hard work, or gained by pure luck.  You invested in trade and nothing was beyond your reach.  Now it's not worth a dime and no money in the world can guarantee your safety and well being.  Hard times are ahead, so it's time to undust your skills and try to sell some ice to the Innuit.  What currency do they accept?",
+    "points": 2,
+    "cash": 10000000000,
+    "traits": [ "PROF_MILLIONAIRE" ],
+    "skills": [ { "level": 3, "name": "barter" }, { "level": 2, "name": "speech" } ],
+    "items": {
+      "both": {
+        "items": [
+          "suit",
+          "skinny_tie",
+          "tieclip",
+          "socks",
+          "dress_shoes",
+          "smart_phone",
+          "money_bundle",
+          "gold_watch",
+          "gold_ear",
+          "gold_bracelet",
+          "gasdiscount_platinum",
+          "diamond_ring",
+          "gold_small",
+          "silver_small",
+          "platinum_small"
```

Comment:
I'm not sure millionaires are walking around with piles of gold, silver, and platinum in their pockets.

Reply:
ok but it would be REALLY funny though.

Reply:
They don't, but if I'd be a millionaire I'd have some 'go-bag' with some gold, silver, platinum, and diamonds in case of a war or a crisis, when everything loses value, but people still trust in gold. Unfortunately, zombies only trust in brains, so double the trouble for such an unlucky millionaire.